src/pages/CompanyCourseManagement/components/CourseListItem.jsx

```diff
@@ -0,0 +1,67 @@
+import { useTheme } from "@emotion/react";
+import { Grid, ListItem, ListItemText } from "@mui/material";
+import PropTypes from "prop-types";
+
+function CourseListItem({ courseName, courseCategory, trainingNumber }) {
+  const theme = useTheme();
+
+  return (
+    <Grid item xs={12} lg={6}>
+      <ListItem
+        sx={{
+          height: "100px",
+          background: theme.palette.primary.light,
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "flex-start",
+          justifyContent: "center",
+        }}
+      >
+        <ListItemText
+          variant="body2"
+          sx={{
+            textAlign: "left",
+            paddingLeft: "10px",
+            paddingTop: "5px",
```

Comment:
If the courseCategory is too long it will overlap with the trainingNumber. Please add paddingRight to prevent this.